### PR TITLE
Fix security issues and bump some component versions

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
you may look at each commit to get specific details

### What issues does this PR fix or reference?
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7598 reported by github scan

### CODE REVIEW

File: generator/package.json [hunk 1]
```diff
@@ -54,10 +53,6 @@
     "format-code": "tsfmt -r",
     "tslint-fix": "tslint --fix --project .",
     "tslint": "tslint --project .",
-    "exe": "yarn run exe:alpine && yarn run exe:mac && yarn run exe:linux",
```
Comment (ericwill) [inline generator/package.json:57]: How come these were removed?

File: generator/package.json [hunk 2]
```diff
@@ -54,10 +53,6 @@
...
-    "exe:alpine": "rimraf exe/alpine/ && mkdir -p exe/alpine && nexe dist/yargs.js -t alpine-x64-8.12.0 -o exe/alpine/che-theia",
-    "exe:mac": "rimraf exe/macOS/ && mkdir -p exe/macOS && nexe dist/yargs.js -t mac-x64-8.12.0 -o exe/macOS/che-theia",
-    "exe:linux": "rimraf exe/linux/ && mkdir -p exe/linux && nexe dist/yargs.js -t linux-x64-8.12.0 -o exe/linux/che-theia",
```
Comment (azatsarynnyy) [inline generator/package.json:60]: If we aren't providing the generator as a native binary, then the doc has to be updated correspondingly
https://github.com/eclipse/che-theia/blob/aa38014db4e41afba47d08bbc6450a1fdfc14d1e/generator/CONTRIBUTING.md#native-executable

File: generator/package.json (hunk 1 above)
Comment (benoitf) [reply generator/package.json:57]: hello, it comes from the commit

```
feat(nexe): remove nexe from the generator as now it's built from the theia repository
```

as it seems that now we don't need anymore the native binary as the generator is already provided in che-theia repository

File: generator/package.json (hunk 2 above)
Comment (benoitf) [reply generator/package.json:60]: ok, agreed

maybe let's discuss if it's fine to remove it or not ?
I was the one bringing native binary but then I don't know if we still need it as it's now inside this repo

Comment (azatsarynnyy) [reply generator/package.json:60]: I don't see why one would need Che Theia generator in the form of a native binary. I think it's good enough to provide a single way of installing/running Che Theia generator - as npm "binary" only.

**Comment by benoitf** [comment]
@azatsarynnyy fixed commit with nexe and CONTRIBUTING.md  / native binaries

**Comment by dmytro-ndp** [comment]
crw-ci-test

**Pull request merged**